CHANGES.rst

```diff
@@ -77,6 +77,8 @@ Mosviz
 Specviz
 ^^^^^^^
 
+- Fixed loading additional spectra after creating a subset. [#1205]
+
```

Comment:
probably affects anything with a spectral viewer... does that belong in specviz or a more general category?

Reply:
Perhaps...

Reply:
It is defined under `configs/specviz`... 🤷

Reply:
Are the headers from a code or user-perspective? 🤷